src/test/resources/config/application-integration-test.yml

```diff
@@ -0,0 +1,12 @@
+workspace.integration-test:
+  dataRepoInstanceNames:
+    dev: terra
+  dataRepoSnapshotId:
+    dev: 97b5559a-2f8f-4df3-89ae-5a249173ee0c
+  serviceAccountEmail: william.thunderlord@test.firecloud.org
+  serviceAccountFilePath: rendered/service-account.json
+  wsmEndpoints:
+    workspaces: api/workspaces/v1
+  wsmUrls:
+    dev: https://workspace.dsde-dev.broadinstitute.org/
+#    dev: http://localhost:8080/
```

Comment:
nit: delete if not used

Reply:
Ah, we had this before. This is so that it's easy to toggle the comment on these two lines locally when you want to run integration tests against local, since intellij doesn't expose an easy way to override an environment variable.

Reply:
add a code comment to that effect then? e.g.
```
    dev: https://workspace.dsde-dev.broadinstitute.org/
    # toggle to the following value to run integration tests against the local environment
    # dev: http://localhost:8080/
```